### PR TITLE
 start working on my roulette app

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by BogdanNegura — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]

File: src/component/board/board.component.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { StyledBoard } from './board.style'
+import Slot from '../slot'
+
+const Board = () => {
+  return (
+    <StyledBoard side={350}>
```
Comment (tandaraelena) [inline src/component/board/board.component.jsx:7]: Is it "side" or "size"?

File: src/component/slot/slot.style.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+export const StyledSlot = styled.div`
+  height: 175px;
+  position: relative;
+  display: inline-block;
+  margin-left: 165px;
+  transform: rotate(120deg);
+  transform-origin: 50% 100%;
+
+  &::after {
+    content: ${({ number }) => `"${number}"`};
```
Comment (tandaraelena) [inline src/component/slot/slot.style.js:12]: You probably don't need double quotes inside ticks. You can try to remove double quotes and see if works without them.

**Review by tandaraelena — APPROVED** [review]
Left 2 small comments, but overall it's looking good so I approved it. Happy coding !

File: src/component/board/board.component.jsx (hunk 1 above)
Comment (ghost) [reply src/component/board/board.component.jsx:7]: It is "side" :)

File: src/component/slot/slot.style.js (hunk 2 above)
Comment (ghost) [reply src/component/slot/slot.style.js:12]: I'll try now. Thank you, @tandaraelena!

File: src/component/board/board.component.jsx (hunk 1 above)
Comment (tandaraelena) [reply src/component/board/board.component.jsx:7]: @marianzburlea we probably did a mistake naming this property "side" :-?

File: src/component/slot/slot.style.js (hunk 2 above)
Comment (ghost) [reply src/component/slot/slot.style.js:12]: I tried. Without double quotes or single quotes, the number is not displayed on the triangle

**Pull request merged**